packages/strapi-admin/admin/src/containers/NotFoundPage/index.js

```diff
@@ -28,8 +29,24 @@ export default class NotFound extends React.Component { // eslint-disable-line r
         <h2 className={styles.notFoundDescription}>
           <FormattedMessage {...messages.description} />
         </h2>
-        <Link to="/admin">Back to home page.</Link>
+        <Button
+          label="Back to home page"
+          buttonBackground="back"
+          onClick={(e) => {
+            e.stopPropagation();
+
+            this.props.history.goBack();
+          }}
+        >
+          Back to home page.
```

Comment:
`Button` component does not require content, because it is managed by the `label` prop.